packages/scenes/src/components/VizPanel/VizPanelRenderer.tsx

```diff
@@ -190,6 +193,9 @@ export function VizPanelRenderer({ model }: SceneComponentProps<VizPanel>) {
             onFocus={setPanelAttention}
             onMouseEnter={setPanelAttention}
             onMouseMove={debouncedMouseMove}
+            collapsible={collapsible}
+            collapsed={collapsed}
+            onToggleCollapse={onToggleCollapse ? (collapsed: boolean) => onToggleCollapse(collapsed, model) : undefined}
```

Comment:
```suggestion
            onToggleCollapse={model.onToggleCollapse}
```

would have expected this

Reply:
Updated